### PR TITLE
Jmm/1729 sp_framework fields

### PR DESCRIPTION
**Pull request by JeanMarie-TTS**

### CODE REVIEW

**Review by jperson1 — DISMISSED** [review]

File: backend/schemas/source/base/GAAP.libsonnet [hunk 1]
```diff
@@ -20,5 +20,43 @@ local gaap_results = [
...
+    readable: 'Disclaimer of opinion',
+    tag: 'disclaimer_of_opinion',
+  },
+];
 
-{ gaap_results: gaap_results}
\ No newline at end of file
+{ gaap_results: gaap_results}
+{ sp_framework_basis: sp_framework_basis}
+{ sp_framework_opinions: sp_framework_opinions}
```
Comment (jadudm) [inline backend/schemas/source/base/GAAP.libsonnet:62]:
```suggestion
{ gaap_results: gaap_results,
  sp_framework_basis: sp_framework_basis,
  sp_framework_opinions: sp_framework_opinions,
}
```

**Review by jadudm — APPROVED** [review]
LGTM.

File: backend/static/js/audit-info-form.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+const FORM = document.forms[0];
```
Comment (jadudm) [inline backend/static/js/audit-info-form.js]:
```suggestion
const FORM = document.getElementById('upload-report__form');
```

Comment (jadudm) [inline backend/static/js/audit-info-form.js]:
```suggestion
const FORM = document.getElementById('upload-report__form');
```

**Review by jadudm — APPROVED** [review]
LGTM. Again. GH seems to have been out of sync?

**Pull request merged**